cli/cli/kurtosis_config/resolved_config/kurtosis_cluster_config.go

```diff
@@ -77,21 +86,36 @@ func (clusterConfig *KurtosisClusterConfig) GetClusterType() KurtosisClusterType
 func getSuppliers(clusterId string, clusterType KurtosisClusterType, kubernetesConfig *v2.KubernetesClusterConfigV2) (
 	kurtosisBackendSupplier,
 	engine_server_launcher.KurtosisBackendConfigSupplier,
+	*engine_server_launcher.KurtosisRemoteBackendConfigSupplier,
 	error,
 ) {
+	kurtosisRemoteBackendConfigSupplier := engine_server_launcher.NewKurtosisRemoteBackendConfigSupplier(func() (*generated.KurtosisContext, error) {
```

Comment:
can we name this anonymous function and pass it to the supplier?

Reply:
Good idea, I think I'll even burry it at the contexts-config-store, seems the right place to put it, and it will give us an easy access to the current context when we need it 👍

Reply:
Oh welp, no need actually you can just do `store.GetContextsConfigStore().GetCurrentContext` 👍